### PR TITLE
[WIP][SKILLONOMY-207] Platform updates subscription

### PR DESCRIPTION
**Pull request by dyudyunov**
**Youtrack** https://youtrack.raccoongang.com/issue/SKILLONOMY-207

**Changes:**
* model for subscription emails added
* new button in sysadmin dashboard for downloading subscription emails
* endpoint added
* email validation
* error/success messages added
* translations

**Notes:**
* related to https://github.com/raccoongang/edx-theme/pull/1884

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

File: lms/djangoapps/dashboard/sysadmin.py [hunk 1]
```diff
@@ -310,6 +310,18 @@ def post(self, request):
                     (UserProfile.objects.all().iterator()))
             return self.return_csv('users_{0}.csv'.format(
                 request.META['SERVER_NAME']), header, data)
+
+        elif action == 'download_subscription_emails':
+            from .models import PlatformNewsSubscriptionEmail
```
Comment (OPersian) [inline lms/djangoapps/dashboard/sysadmin.py]: Can we avoid local import here?

Comment (dyudyunov) [reply lms/djangoapps/dashboard/sysadmin.py]: yeap, thanks! I just forgot to move it. Fixed

File: lms/djangoapps/dashboard/sysadmin.py [hunk 2]
```diff
@@ -310,6 +310,17 @@ def post(self, request):
                     (UserProfile.objects.all().iterator()))
             return self.return_csv('users_{0}.csv'.format(
                 request.META['SERVER_NAME']), header, data)
+
+        elif action == 'download_subscription_emails':
+            emails = PlatformNewsSubscriptionEmail.objects.all().values_list('subscription_email')
+            if emails:
+                return self.return_csv('plarform_updates_subscriptions',
```
Comment (drtwisted) [inline lms/djangoapps/dashboard/sysadmin.py]: I guess this method call could fit on a single line. If not, then please, use [these code-style rules](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines).

Comment (dyudyunov) [reply lms/djangoapps/dashboard/sysadmin.py]: fixed, thx

**Pull request merged**